cli/bench_command.go

```diff
@@ -66,305 +62,497 @@ type benchCmd struct {
 	deDuplicationWindow  time.Duration
 	retries              int
 	retriesUsed          bool
+	ack                  bool
 }
 
 const (
-	DefaultDurableConsumerName string = "natscli-bench"
-	DefaultStreamName          string = "benchstream"
-	DefaultBucketName          string = "benchbucket"
+	BenchDefaultDurableConsumerName string = "nats-bench"
```

Comment:
please unexport all these, just `benchDefaultDurableConsumerName` not `BenchDefaultDurableConsumerName`

Reply:
changed all my consts to not be exported